module-infra/src/main/kotlin/org/depromeet/team3/place/PlaceQuery.kt

```
@@ -62,30 +63,30 @@ class PlaceQuery(
+        val placesNeedingUpdate = cachedPlaces.values.filter { entity: PlaceEntity -> 
+            entity.photos.isNullOrBlank() || entity.addressDescriptor.isNullOrBlank()
+        }
+        if (placesNeedingUpdate.isNotEmpty()) {
+
+            // 업데이트가 필요한 장소들을 API로 다시 조회
+            val updateResults = placesNeedingUpdate.map { entity: PlaceEntity ->
                 entity.googlePlaceId?.let { placeId: String ->
                     try {
                         val response = googlePlacesClient.getPlaceDetails(placeId)
                         Triple(response, openNowMap[placeId], linkMap[placeId])
                     } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)